go-controller/pkg/config/config.go

```diff
@@ -653,7 +653,13 @@ func PrepareTestConfig() error {
 	ClusterManager = savedClusterManager
 	Kubernetes.DisableRequestedChassis = false
 	EnableMulticast = false
-	Default.OVSDBTxnTimeout = 5 * time.Second
+
+	// OVSDBTimeout can be overriden for test, reduce to reduce test exection time
+	// increase to reduce flakiness, if unset or parsing failed, defaults to 5s
+	Default.OVSDBTxnTimeout, _ = time.ParseDuration(os.Getenv("TEST_OVSDB_TXN_TIMEOUT"))
+	if Default.OVSDBTxnTimeout <= 0 {
+		Default.OVSDBTxnTimeout = 5 * time.Second
```

Comment:
why don't you just set it to 1s here without extra env variables?

Reply:
My first reaction was just set it, right. But this timeout is used in different tests, and it used to verify asynchronous behavior. I thought it can be tuned for runners as tuned are, usually, not as powerful as local dev env.

But new env variable for sure creates configuration overhead for tests. But as it only used it tests so I thought it would be ok. 

So if you feel we need just lower the value I will revert env variable changes and will just reduce value to sensible default (1s seems to work on GitHub runner).